src/main.rs

```diff
@@ -39,11 +38,10 @@ struct Config {
 struct JobItem {
     language: Option<LANG>,
     path: PathBuf,
-    cfg: Config,
+    cfg: Arc<Config>,
 }
 
 type JobReceiver = Receiver<Option<JobItem>>;
```

Comment:
I don't really see the point to remove JobSender alias but ok: it isn't super used.
For consistency, you should remove JobReceiver too.

Reply:
My bad. The code is clearer having it, thanks for spotting that! Fixed!